docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,54 +1,29 @@
 
-.. Select *Experience* from the drop-down list, add the title *User should be able to launch applications through a simple work-flow*, and Press kbd:[Enter].
-
-. Create an execution oriented child *Feature* for the *Experience* using the in-line add option with the title *User should be able to easily launch a sample Vert.x application using a smooth developer work-flow*.
-
-. Create the following child *Task* work items to implement the *Feature* using the in-line add option:
-
-* <<creating_new_project-hello-world,*Launch a sample quickstart application*>>
-* <<approving_build_pipeline,*Test the application in Stage and approve to the Run environment*>>
-* <<creating_che_workspace-hello-world,*Modify the quickstart codebase*>>
-* <<reviewing_publishing_changes-hello-world,*Review and publish changes to your codebase*>>
+. Using the in-line add option create an execution oriented child *Story* for the *Epic* with the title *User should be able to easily launch a sample Vert.x application* and a child *Task* for the story titled *Review and publish changes to your codebase*.
 +
-image::gs_tasks_egs.png[Example Tasks]
+See link:user-guide.html#working_with_work_items[working with work items] for more information about modifying work items.
 
-. Similarly, select *Fundamental* in the *Add Work Item* pane, to create a fundamental work item with the title *The application should be secure with appropriate licenses* and use the in-line add option to create a child experience, feature, and task for it. The task can be titled <<analytics_stack_report,*Use analytics to identify and resolve security and license issues in the application*>>.
-+
-image::gs_fundamental_task_eg.png[Example Fundamental Task]
-+
-. In the left panel, click the (+) icon adjacent to *Iterations* to link:user-guide.html#creating_a_new_iteration[create an iteration] called *Test Iteration* and force activate it.
-. Click on each of the feature and child task work items to preview it, and:
-
-* link:user-guide.html#tracking_state_of_a_work_item[Change the state of the work items] to *In Progress* when you start working on them
-*  link:user-guide.html#assigning_the_work_item[Add yourself as the assignee]
-* link:user-guide.html#associating_work_items_with_an_iteration-user-guide_iterations[Associate the work items to the *Test Iteration*]
-+
-See link:user-guide.html#working_with_work_items[working with work items] for detailed information about modifying and tracking work items by associating them to iterative development cycles.
+. In the left panel, click the (+) icon adjacent to *Iterations* to link:user-guide.html#creating_a_new_iteration[create an iteration] called *Test Iteration*, force activate it, and  link:user-guide.html#associating_work_items_with_an_iteration-user-guide_iterations[associate the story and task to the *Test Iteration*].
```

Comment:
This should also be broken down into multiple steps that are easy to follow along. I'm not sure we need an iteration for this example to start with, TBH. I don't know why we're directing users to a separate document for associating the story and task, this should be elaborated here as well.

Reply:
@mishaone this was the case earlier, but it adds to the steps and substeps. This was changed based on @rkratky  inputs that this module is very long, we had 8 steps with many of the steps having substeps. Let me know if we want substeps again
Well Iteration is the way you will track the WIs in the development cycle. 
There was a screenshot along with expalnation for the same, I removed it since we wanted to cut down on screenshots and also number of steps. Please see the current prod version of our docs. 
Seems like I am going back and forth on these. I suggest we all meet sometime to discuss exactly how we want to go about this module. I will note down the AIs from there and work on it.

Reply:
I don't think the best way to address longer modules is to skip or merge steps, tbh. Maybe we can reduce the things we're doing so that it's simpler and shorter instead somehow?

Reply:
>I don't think the best way to address longer modules is to skip or merge steps, tbh.
I agree, but with the current intent and the feedback to cut down length this and links to the detailed docs was what I thought of.
I am open to suggestions, we have our team meeting on Tuesday, I suggest we discuss then or if we can have it earlier that would be great too. Discussing this here is really not working out.

Reply:
I don't honestly know that we can be much help given that you're the "expert" on planner so to speak. I think the way forward is to identify a more streamlined and minimalist sort of workflow that suits the context here. I don't think there's much I can say about the specifics of that since I don't know much about the intended workflow you have at the moment or what the best thing to show users at this point is. Maybe Robert can say something about that?

Reply:
My expertise or lack of it is not the point, this one is for our docs team to decide on. @mishaone my aim for the discussion would be 1. finalize on the intent: Tying disjoint actions or not, 2. I could explain the 2 workflows we can have, minimal or slightly deeper. I understand your viewpoint, I started with a different one. I just want a clarity on which one we as a team want to follow. I am happy to go along what ever all of us decide.
I don't see why we are so hesitant on meeting on this. We are spending a fair bit of time writing back and forth for what could be resolved in 1 meeting. AFAIR a meeting was able to resolve 2 issues we were stuck on earlier fairly quickly.

Reply:
I mean, once again we're at a crossroads with two opinions. I think this can be relatively easily and quickly decided on via email or comments. I thought we had already decided on a more streamlined and basic workflow for this document. I don't know much about the planner, as I said, so I'm not sure what I can contribute to this aside from what I've already contributed. I'm happy to meet but unsure of the benefits in this particular instance.